Adds inverted 4x1 and 4x2 widgets to apollo

travp624 opened the pull request:
we all love black background with white text and seeing as themes can not change the widget. This adds a 4x1 and 4x2 inverted widget. 

Also couldn't figure out how to only send the one commit I don't get why the other one is attached its for building in source cm style. But I couldn't remove it.

rmcc committed:
Make Apollo build with the platform

Commit:
CM9-Apollo-Added Russian Translation

CM9-Apollo-Update Russian Translation

Gustav0ar committed:
PT-BR: Initial translations

Change-Id: I476e6caddf6243c1e9deb226d88dd032c233f4ee

therbom committed:
Apollo: Dutch translations

Change-Id: I82547c1adfbc45794fa6d0b5937a881b4a63a8a6

adneal committed:
updated missing drawables

adneal committed:
Use notifyDataSetChanged() in place of invalidateViews() and cleared Lint warnings

Commit:
Added two new widgets to apollo widgets
-a inverted 4x2 widget
-a inverted 4x1 widget

Commit:
Correct when a theme calls for this style that it inverted the whole app.

Commit:
darken the divider for inverted widgets.

Commit:
Inverted Apollo cause the white is killing me.

Commit:
revert "Inverted Apollo cause the white is killing me." (reverse-merged from commit 522be87227243b6e11b354068bb9d932db86f7d1)

Commit:
revert "Correct when a theme calls for this style that it inverted the whole app." (reverse-merged from commit f82cd186519eb3476663662c5391882e58e07dd5)

travp624 commented:
What ever happened to just sending one pull request of only one commit now everything I keep pushing to my branch gets added to this what the heck.

mabako commented:
You made it a pull request of master. Everything you add to your master is added here :p You should create a new branch for each pull request.

Commit:
CM9-Apollo-Update Russian Translation

CM9-Apollo-Update Russian Translation

mabako committed:
German localization for Apollo

Added translatable="false" to key_-Constants.

Change-Id: Ibec71339f374c748f651635540a309d9c1232e23

mabako committed:
German localization for Apollo

Added translatable="false" to key_-Constants.

Change-Id: Ibec71339f374c748f651635540a309d9c1232e23

travp624 commented:
commits I made even after submitting the pull request are getting added to this I don't remember it working that way before Of course I'm not a github guru or anything so.

adneal commented:
Make sure you use the **Holo** **appwidget_bg** and rather than naming it "**Inverted Apollo**..." I'd prefer it be "**Apollo Inverted**...". If I can't add theme capability for the AppWidgets, then I'll merge it.

Commit:
Change inverted 4x1 and 4x2 widgets to "Apollo inverted *"

travp624 commented:
Changed the naming on the widgets. Wish I would have figured out how to just include the right commits here. But I know you'll get the jist of it if needed. If this can be done eventually in a theme I guess that would be much simpler yes.

takuo committed:
Japanese: initial translation for Apollo.

Change-Id: I2b272dec40912b0db912cee029dc1da4adfa96af

AndroidMeda committed:
Apollo: Turkish translations

Change-Id: Iaf6d6c71e7db9a97902268f8c6ee6f0011aa44b7

AndroidMeda committed:
Apollo: typo

Change-Id: I6de875bfd99b3650b8909728a69730b77b8e6df5

Fra-90 committed:
Apollo - Update Italian Translation Thanks To Moreno Pojer

Change-Id: I9761a16db119624518059c1b6ace216dc7c3a817

Commit:
Added localization for Apollo music player tabs
Translated Apollo to spanish language

This adds capability for translate some parts of Apollo music player
Patch-set 2: Restored attribute translatable="false"

Change-Id: Ib453d28901bb21ade0055487cdb33e5e052d5a93

epinter committed:
Apollo: PT-BR translations

Added PT-BR translations for tabs.

Change-Id: I9168f95769b05ab0da63e7959cc06090a9e8af54

Tserppa committed:
Apollo: Finnish translation

Change-Id: I90f2b465e2478cd1085c80a4a8a7b121b3f3cbb8

mabako committed:
Apollo: Made "Constants" a class, use static imports instead of "implements"

Aside from that, a few strings are now static so they're not getting reinstated
with every single class; some classes didn't use any Constants but did
nevertheless waste memory for that.

Patch Set 2: Fixed a tab -> space

Change-Id: I0edb082ecd9c50f85ea0380d6b5eba9f9c469415

mabako committed:
Apollo: fix navigation behavior for selecting an artist -> album

From navigating to any artist, then any item, the previous behaviour handles
the "back" button by skipping the artist view and returning directly to the
main view (list of artists).

This patch changes this, in that the "back" button works exactly the same as
the "up" button (arrow in the actionbar) for when you follow the route
mentioned above.

Change-Id: I73055bf79ec54a51af7a6026f5a0191819e8e2b7

Commit:
Catalan translation

Change-Id: I1c6443d2aeac23f62df7f48f7dcfe14a07bc4dbb

bugadani committed:
Apollo Hungarian translations

Change-Id: Ia33a8b0e03be395ca379d02ef01456fbaa611f11

tpruvot committed:
french: initial translation

Change-Id: I8fe363af4cf8c19c5ffbc59132d4a282edef79e4

alankila committed:
Fix font rendering on listview items

The small shadow layer around the fonts seems to cause some kind of issue where
the antialiasing looks extremely poor, sa if the rendering was replete with gamma errors.

Change-Id: I9c6be758b8f1faa1fe84ec0252a8570d89c5de80

therbom committed:
Apollo: Dutch translations

Change-Id: I4c7f9dc8365978159dc22c6f3e80a48fc274b805

AndreasBoehm committed:
Apollo: Update german translation
Signed-off-by: Andreas Böhm <andreas@boehm.cx>

Change-Id: I746f9312ddcf9e283d4c1e34b440ec18530d81ba

Commit:
Added Hebrew Translation

Change-Id: I4b9dc83a723509e5cc11dde0449efee1e4e2e760

travp624 committed:
fix inverted widgets for constants
